src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -196,7 +197,9 @@ func (q *qemuAmd64) cpuModel() string {
 }
 
 func (q *qemuAmd64) memoryTopology(memoryMb, hostMemoryMb uint64, slots uint8) govmmQemu.Memory {
-	return genericMemoryTopology(memoryMb, hostMemoryMb, slots, q.memoryOffset)
+	memory := genericMemoryTopology(memoryMb, hostMemoryMb, slots, q.memoryOffset)
+	memory.MemoryModules = genericNUMAMemoryModles(memoryMb, 4, q.numaNodes)
+	return memory
```

Comment:
cc @kevinzs2048 @BbolroC @Amulyam24 to check what should be done (if something should be done) for aarch64 / s390x / ppc64le.

Reply:
Ok than lets first enable it for x86 and arm64 and follow up with ppc64 if needed!